src/ctxmenu.ts

```diff
@@ -134,6 +139,10 @@ class ContextMenu implements CTXMenuSingleton {
         this._hide(this.menu);
     }
 
+    public setNonce(nonce: string): void {
+        this.nonce = nonce;
```

Comment:
when you make `nonce` a static member (see below) this needs to change to `ContextMenu.nonce = nonce`